test/unit/amp/amp-story-page-handler.js

```diff
@@ -263,9 +256,70 @@ describe("ampStoryPageHandler unit tests", function () {
       ...dummyOpts,
     });
     assert.strictEqual(getAdditionalConfigCalled, true);
-    assert.strictEqual(additionalConfigReceivedByAmplib, JSON.stringify({
-      key111: 'this contains bk config by default',
-      key222: 'this is the fetched additional config'
-    }))
+    assert.strictEqual(
+      additionalConfigReceivedByAmplib,
+      JSON.stringify({
+        key111: "this contains bk config by default",
+        key222: "this is the fetched additional config",
+      })
+    );
+  });
+  it("Should render timezone if it's passed to seo if timezone is present in publisher config", async function () {
+    let seoPassedToAmpLib;
+    let timezone = null;
+    const dummySeo = (config, pageType) => {
+      return {
+        getMetaTags: (config, pageType, data = {}, url) => {
+          timezone = data.data.timezone;
+          return {
+            toString: () => "2021-07-15T11:35:20.008+05:30",
+          };
+        },
+      };
+    };
+    const dummyAmpLib = {
+      ampifyStory: (params) => {
+        seoPassedToAmpLib = params.seo;
+      },
+      unsupportedStoryElementsPresent: () => false,
+    };
+    await ampStoryPageHandler(dummyReq, dummyRes, dummyNext, {
+      client: getClientStub(),
+      config: dummyConfig,
+      domainSlug: null,
+      seo: dummySeo,
+      additionalConfig: { publisher: { timezone: "Asia/Kolkata" } },
+      ampLibrary: dummyAmpLib,
+      InfiniteScrollAmp: DummyInfiniteScrollAmp,
+    });
+    assert.strictEqual(timezone, "Asia/Kolkata");
+    assert.strictEqual(seoPassedToAmpLib, "2021-07-15T11:35:20.008+05:30");
+  });
+  it("Should render null if it's not passed to seo if timezone is not present in publisher config", async function () {
```

Comment:
and this to `should call seo lib with null timezone if timezone info is absent in publisher config`